docs/user-guide/app_dependencies.md

```diff
@@ -6,6 +6,12 @@ Using the new dependency mechanism, users are not bound to the app-of-apps patte
 
 Application dependencies support both, manual and automatic sync policies.
 
+## Sync behaviour
+
+Defining dependencies for an Application will affect that Application's sync behaviour. When a sync is started, Argo CD will evaluate the sync and health status of all other Applications in the dependency tree. Only if all of the other Applications that the Application is dependent upon have a sync status of `Synced` and a health status of `Healthy`, the sync will proceed. Until all conditions are met, the sync will stay in a progressing state and a message will indicate that the sync is waiting for dependencies. This behaviour is similar to sync waves, where a sync progresses only to the next wave when the current wave's resources are all synced and healthy.
```

Comment:
small suggestion:
`s/affect that Application's/affect the Applications's`

Reply:
I'm on the fence on this.

Reply:
```suggestion
Defining dependencies for an Application will affect its sync behaviour. When a sync is started, Argo CD will evaluate the sync and health status of all other Applications in the dependency tree. Only if all of the other Applications that the Application is dependent upon have a sync status of `Synced` and a health status of `Healthy`, the sync will proceed. Until all conditions are met, the sync will stay in a progressing state and a message will indicate that the sync is waiting for dependencies. This behaviour is similar to sync waves, where a sync progresses only to the next wave when the current wave's resources are all synced and healthy.
```

@leoluz @jsoref How about this one instead?

Reply:
```suggestion
Defining dependencies for an Application will affect its sync behaviour. When a sync is started, Argo CD will evaluate the sync and health status of all other Applications in its dependency tree. Only if all of its dependent Applications have a sync status of `Synced` and a health status of `Healthy`, will the sync proceed. Until all conditions are met, the sync will stay in a progressing state and a message will indicate that the sync is waiting for dependencies. This behaviour is similar to sync waves, where a sync progresses only to the next wave when the current wave's resources are all synced and healthy.
```